src/test/resources/cft/AWS-SNS-subscription-is-not-configured-with-HTTPS/AWS-SNS-subscription-is-not-configured-with-HTTPS-negative.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
This test case for S3 bucket and not for SNS Subscription. Pls remove this one and add one for SNS.